buildpipeline/pipeline.json

```diff
@@ -98,7 +98,7 @@
             "PB_DistroRid": "ubuntu.14.04-arm", 
             "PB_DockerTag": "ubuntu-14.04-cross-0cd4667-20172211042239", 
             "PB_TargetArchitecture": "arm", 
-            "PB_AdditionalBuildArguments":"arm cross disablecrossgen -portable skiptests", 
+            "PB_AdditionalBuildArguments":"-TargetArchitecture=arm -DistroRid=linux-arm -DisableCrossgen=true -PortableBuild=true -SkipTests=true",
```

Comment:
I looked at how Jenkins was actually invoking the build (based on what build.sh was computing) and moved it here.